app/index.js

```diff
@@ -579,13 +579,21 @@ ipcMain.on("download-list-request", (e, action, downloadId) => {
 ipcMain.on("set-permissions", (_, permissionObject) => {
     permissions = permissionObject
 })
-ipcMain.on("set-spelllang", (_, lang) => {
+ipcMain.on("set-spelllang", (_, langOptions) => {
+    langOptions = new Set(
+        langOptions.split(",").map(lang => {
+            if (lang === "system") {
```

Comment:
I like this change, seems much more accurate than setting an empty list (which worked on some systems, but also resulting in no language being checked on others).